test/e2e/storage/csi_mock_volume.go

```diff
@@ -1988,3 +2107,35 @@ func getVolumeLimitFromCSINode(csiNode *storagev1.CSINode, driverName string) in
 	}
 	return 0
 }
+
+// checkDeleteSnapshotSecrets checks if delete snapshot secrets annotation is applied to the VolumeSnapshotContent.
+func checkDeleteSnapshotSecrets(cs clientset.Interface, annotations interface{}) error {
+	ginkgo.By("checking if delete snapshot secrets annotation is applied to the VolumeSnapshotContent")
+
+	var (
+		annDeletionSecretName      string
+		annDeletionSecretNamespace string
+		ok                         bool
+		err                        error
+
+		// CSISnapshotterSecretName is a parameter to be provided for VolumeSnapshotClass.parameters
```

Comment:
CSISnapshotterDeleteSecretNameAnnotation is the annotation key for the CSI snapshotter delete secret name in VolumeSnapshotClass.parameters